merkletree/tb.go

```diff
@@ -2,12 +2,20 @@ package merkletree
 
 import "github.com/coniks-sys/coniks-go/crypto/sign"
 
+// TemporaryBinding is a protocol extension which
+// serves two main functions: (1) as a signed promise
+// by the key server to include a newly registered
+// (or updated key) in the next epoch, and (2) to allow immediate
+// use of the key without having to wait for the next epoch.
```

Comment:
As I mentioned before, I wouldn't use protocol-related terminology here. I think it's fine to say "PAD" instead of "key server" and "inserted" instead of "registered". Also, the use of "key" is ambiguous here, too.